codegen/src/main/twirl/templates/ScalaClient/Client.scala.txt

```diff
@@ -96,13 +96,18 @@ final class @{service.name}Client(settings: GrpcClientSettings)(implicit mat: Ma
   @if(methodType == akka.grpc.gen.BidiStreaming) {MethodDescriptor.MethodType.BIDI_STREAMING }
 }
 
-object @{service.name}Client {
+object @{service.name}Client { clientObj =>
 
-  import @{service.name}.Serializers._
+  implicit val factory: AkkaGrpcClientFactory[@{service.name}Client] = new AkkaGrpcClientFactory[@{service.name}Client] {
```

Comment:
The factory is added an implicit value in the gRPC's client's companion object. We couldn't make the companion into a factory because it causes an `unrecoverable cycle resolving import` compile error.

Reply:
Yes that seems simpler, let's go for it.